src/plugins/dashboard/public/application/legacy_app.js

```diff
@@ -165,7 +165,7 @@ export function initDashboardApp(app, deps) {
           };
 
           $scope.editItem = ({ editUrl }) => {
-            history.push(deps.addBasePath(editUrl));
+            history.push(editUrl);
```

Comment:
Not sure if this is a good idea :

Dashboard's own registration is :
```
     viewUrlPathFn: (obj) => `#/view/${obj.id}`,
      editUrlPathFn: (obj) => `/view/${obj.id}?_a=(viewMode:edit)`,
```

Observability's registration is. :
```
      editUrlPathFn: (obj: SavedObject) => `/app/observability-dashboards#/${obj.id}/edit`,
      viewUrlPathFn: (obj: SavedObject) => `/app/observability-dashboards#/${obj.id}`,

```

Will both of these work with straight `history.push(editUrl)` ?

Reply:
@ashwin-pc @abbyhu2000